parsers/promotedTitle.js

```diff
@@ -45,15 +38,14 @@ function getPromotedTitle(snippet) {
 	});
 
 
-	if (title === "" || title == undefined) {
-		debug("#" + postcount + ": title [" + snippet._id + "] NOT FOUND");
-		error = 1;
+	if ( _.size(title) === 0 || !title ) {
```

Comment:
* if you use !title it match when title is `null` `undefined` `0` or `NaN`
  * debug takes format("%s", string); 
  * when something is not found, I use to return `false`, so we can isolate such cases
  * instead of using `_id` of Mongo, use the `id`, because it is ours and can be called in our API to re-iterated the test if necessary